src/Trace/Integrations/PDO.php

```diff
@@ -31,16 +31,20 @@
  */
 class PDO implements IntegrationInterface
 {
+    public static $serviceName = '';
+
     /**
      * Static method to add instrumentation to the PDO requests
      */
-    public static function load()
+    public static function load($serviceName='')
```

Comment:
1. We should have a contrib repo, where we have these new/v2/modified integrations impl. For this we should not fork the whole lib i.e. census-instrumentation/opencensus-php.
2. This should implement IntegrationInterface, changing method signature breaks it.
3. Service name should not be needed at this layer. Need to understand.

Reply:
Service name, if what I understand it is, is captured as part of trace init as well.

Reply:
Also what is the diff between census-instrumentation/opencensus-php and nenad/opencensus-php?